### ISSUE TITLE
images.rapgenius.com shouldn't be blocked

### ISSUE THREAD

**Issue by terrorist96** [EFForg/privacybadgerchrome#824]
http://genius.com/Eminem-not-afraid-lyrics

`images.rapgenius.com` needs to be yellow; otherwise it breaks images.

**Comment by terrorist96** [comment]
So other than clear cookies related to rapgenius, what do I need to do to get PB to realize that it shouldn't be blocking images.rapgenius.com?
It's blocked on my main profile, but allowed on my test profile.

**Comment by cooperq** [comment]
Okay, This should be completely fixed in the latest version of privacy badger. As of today I have only deployed it to 10% of users (it has some huge changes!) but if you want to try it out early you can take the following steps. 
1. Download the crx file [from the chrome web store] (https://clients2.googleusercontent.com/crx/blobs/QgAAAC6zw0qH2DJtnXe8Z7rUJP1rX98iOE5SwxVXgdVTMTKqobiCFyOuS_jNFoVMdRhW-bEA7QhvVJw2hrKrZS8b8ThMYsv0t-wxxLr4D-Sm2ZC1AMZSmuW2zjq5MsW529CVI2Rtq2QO7xWEQg/pkehgijcmpdhfbdbbnkijodmdjhbjlgp_main.crx)
2. Open a new tab to [chrome://extensions](chrome://extensions)
3. Drag the downloaded file and drop it onto the extensions window. 
You should now have Privacy Badger version 2016.5.15
If this doesn't fix the problem please let me know. If you find any new bugs please file a separate issue.

**Comment by terrorist96** [comment]
I'll wait for the full release. :)
When do you expect it to be pushed to 100% of your users?

**Comment by cooperq** [comment]
By the end of this week.

**Comment by terrorist96** [comment]
@cooperq I'm sorry to say, but I got the May 16 update and images.rapgenius.com is still redlisted. I tried clearing cookies as well.
